XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Runtime/Utilities/WindowsApiChecker.cs

```diff
@@ -1,65 +1,74 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using UnityEngine;
+#if WINDOWS_UWP
+using Windows.Foundation.Metadata;
+#elif (UNITY_WSA && DOTNETWINRT_PRESENT)
```

Comment:
The DOTNETWINRT, is this using the .net scripting backend? I thought we agreed to only target the il2cpp scripting backend

Reply:
No we are not using it. Forgot to remove this. Will do.

Reply:
This is resolved.